O/OpenBLAS/OpenBLAS@0.3.26/build_tarballs.jl

```diff
@@ -7,7 +7,7 @@ name = "OpenBLAS"
 version = v"0.3.26"
 
 sources = openblas_sources(version)
-script = openblas_script(;aarch64_ilp64=true, num_64bit_threads=512)
+script = openblas_script(;aarch64_ilp64=true, num_64bit_threads=512, bfloat16=true)
```

Comment:
According to https://github.com/OpenMathLib/OpenBLAS/issues/3403#issuecomment-1891099091, we should also be able to reduce the number of threads we build with and significantly reduce the memory utilization. I wonder if we can build that into this.

Reply:
What is a good default to put in here now? Do we quarter this and do 128, or even lower?

Reply:
If we change it - let's only do it for 0.3.26 so we can test things out with 1.11-dev. It may be disruptive on 0.3.23.